src/app/app.component.html

```diff
@@ -1,4 +1,5 @@
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <img src="assets/branding/img/logo.png" width="200" height="40"/>
```

Comment:
This assumes a fixed size for the logo image (and a bit of an odd aspect ratio at 5w:1h) - worth documenting in the README for how to deploy a new trial.

Reply:
We could add css to get around this when we start deploying per-trial styles as part of the branding, rather than directly sizing the element